src/cast-player.js

```diff
@@ -101,43 +102,31 @@ class CastPlayer extends BaseRemotePlayer {
   /**
    * Loads a media to the receiver application.
    * @param {ProviderMediaInfoObject} mediaInfo - The entry media info.
-   * @param {Object} [options] - The request options. See {@link https://developers.google.com/cast/docs/reference/chrome/chrome.cast.media.LoadRequest|chrome.cast.media.LoadRequest}
+   * @param {Object} request - The cast load request options.
    * @returns {Promise<void>} - Promise to indicate load succeed or failed.
    * @instance
    * @memberof CastPlayer
    */
-  loadMedia(mediaInfo: Object, options?: Object): Promise<*> {
-    this._logger.debug('Load media', mediaInfo, options);
```

Comment:
Is this change breaking current logic? when we called loadMedia api we used to:
1. Reset player and UI
2. optionally dispatch `CHANGE_SOURCE_STARTED` 
3. create cast request object
4. call cast loadMedia

Now we only do 4 and api has changed as we get request instead of [options]

The api is called from external so I don't see how this is handled

Reply:
The api is called from external? 
currently it called by `play` and `_setupRemotePlayer`